src/ocean/text/xml/Document.d

```diff
@@ -1204,7 +1205,7 @@ else
                         else
                            if (nextSibling)
                               {
-                              debug assert(host.firstChild == this);
```

Comment:
Those were `debug` tho

Reply:
Yep, and thus almost never really run. I think they are simple and useful enough to keep them as verifies.